lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4926,22 +4924,22 @@ LogicalResult StmtEmitter::emitVerifAssertLike(Operation *op, Value property,
 }
 
 LogicalResult StmtEmitter::visitVerif(verif::AssertOp op) {
-  return emitVerifAssertLike(op, op.getProperty(), PPExtString("assert"));
+  return emitVerifAssertLike(op, PPExtString("assert"));
 }
 
 LogicalResult StmtEmitter::visitVerif(verif::AssumeOp op) {
-  return emitVerifAssertLike(op, op.getProperty(), PPExtString("assume"));
+  return emitVerifAssertLike(op, PPExtString("assume"));
 }
 
 LogicalResult StmtEmitter::visitVerif(verif::CoverOp op) {
-  return emitVerifAssertLike(op, op.getProperty(), PPExtString("cover"));
+  return emitVerifAssertLike(op, PPExtString("cover"));
 }
 
 /// Emit an assert-like operation from the `verif` dialect. This covers
 /// `verif.clocked_assert`, `verif.clocked_assume`, and `verif.clocked_cover`.
-LogicalResult StmtEmitter::emitVerifClockedAssertLike(
-    Operation *op, Value property, Value disable, Value clock,
-    verif::ClockEdge edge, PPExtString opName) {
+template <typename Op>
```

Comment:
Again, there is a code-size tradeoff doing this instad of just taking the arguments.